ibis/tests/expr/test_table.py

```diff
@@ -674,22 +719,23 @@ def test_aggregate_keywords(table):
     assert_equal(expr2, expected)
 
 
-def test_filter_aggregate_pushdown_predicate(table):
-    # In the case where we want to add a predicate to an aggregate
-    # expression after the fact, rather than having to backpedal and add it
-    # before calling aggregate.
-    #
-    # TODO (design decision): This could happen automatically when adding a
-    # predicate originating from the same root table; if an expression is
-    # created from field references from the aggregated table then it
-    # becomes a filter predicate applied on top of a view
+# TODO(kszucs): revisit this test case, but this usage should not be allowed
```

Comment:
Does this still need to be revisited?

Reply:
We don't eagerly rewrite anymore because that would break column dereferencing entirely. So I'm going to remove this test case. We can do the same using the rewrites BTW.